src/declarations/stencil-public-compiler.ts

```diff
@@ -2028,8 +2028,15 @@ export interface OutputTargetBaseNext {
  *
  * - `default`: No additional export or definition behavior will happen.
  * - `single-export-module`: All components will be re-exported from the specified directory's root `index.js` file.
+ * - `auto-define-custom-elements`: Enables the auto-definition of a component and its children (recursively) in the custom elements registry. This
+ * functionality allows consumers to bypass the explicit call to define a component, its children, its children's
+ * children, etc. Users of this flag should be aware that enabling this functionality may increase bundle size.
  */
-export const CustomElementsExportBehaviorOptions = ['default', 'single-export-module'] as const;
+export const CustomElementsExportBehaviorOptions = [
+  'default',
+  'single-export-module',
+  'auto-define-custom-elements',
```

Comment:
I'm gonna hold off on merging until we answer https://github.com/ionic-team/stencil-site/pull/916#discussion_r974243770, in the event we do wanna change the doc order _and_ if we wanna propagate those changes here